source/_posts/2020-02-26-release-106.markdown

```diff
@@ -254,7 +254,7 @@ Experiencing issues introduced by this release? Please report them in our [issue
 
 - **UniFi** - Changing tracking settings will no longer disable entities but will remove them completely from the state machine and entity registry. The SSID filter will now mark all wireless clients as `not_home` if they are connected to SSIDs that are not a part of the filter. - ([@Kane610] - [#31762]) ([unifi docs])
 
-- **Rest** - Many devices continue to use XML for RESTful APIs. This change implements converting XML output to JSON via `xmltojson` so it can work with the existing rest sensor component. As the attributes that usually need to be scraped are deeper in the document, support for passing in a template to find the JSON attributes has been added. JSON APIs that do not have their attributes at the top level can also benefit from this change. - ([@bdraco] - [#31809]) ([rest docs])
+- **Rest** - Many devices continue to use XML for RESTful APIs. This change implements converting XML output to JSON via `xmltojson` so it can work with the existing rest sensor component. `xmltojson` tags may be confusing; if you are not familiar with the conversion, please use the [xmltodict](https://xmltodict-debugger.glitch.me/) tool created by @balloob to identify the tags to be used to parse the resulting json. As the attributes that usually need to be scraped are deeper in the document, support for passing in a template to find the JSON attributes has been added. JSON APIs that do not have their attributes at the top level can also benefit from this change. - ([@bdraco] - [#31809]) ([rest docs])
```

Comment:
```suggestion
- **Rest** - Many devices continue to use XML for RESTful APIs. This change implements converting XML output to JSON via `xmltojson` so it can work with the existing rest sensor component. 

`xmltojson` tags may be confusing; if you are not familiar with the conversion, please use the [xmltodict](https://xmltodict-debugger.glitch.me/) tool created by [@balloob] to identify the tags to be used to parse the resulting JSON.

As the attributes that usually need to be scraped are deeper in the document, support for passing in a template to find the JSON attributes has been added. JSON APIs that do not have their attributes at the top level can also benefit from this change.

- ([@bdraco] - [#31809]) ([rest docs])
```